spec/rspec/abq_spec.rb

```diff
@@ -6,6 +6,13 @@ def stringify_keys(hash)
 end
 
 RSpec.describe RSpec::Abq do
+  before do
```

Comment:
I would have expected this to be enough to remove flakiness but ... apparently it wasn't 😢